server/app/genbank.py

```diff
@@ -102,3 +112,96 @@ def convert_gbk_to_vector(genbank_file, level: VectorLevel) -> GenbankData:
         features=features,
         references=references,
     )
+
+
+# Converts a model.Vector to a Genbank output
+def convert_LevelN_to_genbank(
+    vector: model.Vector, database: Session = Depends(deps.get_db)
```

Comment:
You're not using your `database` parameter.
```suggestion
    vector: model.Vector
```